source/usage-examples/deleteOne.txt

```diff
@@ -0,0 +1,97 @@
+.. _csharp-find-multiple:
+
+=================
+Delete a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can delete a document from a ``MongoCollection`` by using the synchronous
+`DeleteOne() <{+api-root+}/Overload_MongoDB_Driver_IMongoCollectionExtensions_DeleteOne.htm>`__
+method, or the asynchronous `DeleteOneAsync() <{+api-root+}/Overload_MongoDB_Driver_IMongoCollectionExtensions_DeleteOneAsync.htm>`__ method. Both methods use a **query filter** which specifies the search criteria.
+
+Examples
+--------
+
+These examples use the following ``Restaurant`` class as a model:
+
+.. literalinclude:: ../includes/code-examples/DeleteOneAsync.cs
+   :start-after: start-model
+   :end-before: end-model
+   :language: csharp
+   :copyable:
+   :dedent:
+
+Delete a Document Using Builders
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: builders-async
+
+      The following example uses builders to asynchronously delete a document in 
+      the ``restaurants`` collection with the ``cuisine`` "Continental".
+
+      .. literalinclude:: ../includes/code-examples/DeleteOneAsync.cs
+         :start-after: start-delete-one-builders-async
+         :end-before: end-delete-one-builders-async
+         :language:  csharp
+         :copyable:
+         :dedent:
+
+   .. tab:: Synchronous
+      :tabid: builders-sync
+
+      The following example uses builders to synchronously delete a document in 
+      the ``restaurants`` collection with the ``cuisine`` "Continental".
+
+      .. literalinclude:: ../includes/code-examples/DeleteOneSync.cs
+         :start-after: start-delete-one-builders-sync
+         :end-before: end-delete-one-builders-sync
+         :language:  csharp
+         :copyable:
+         :dedent:
+
+Expected Result
+~~~~~~~~~~~~~~~
+
+Running the :ref:`full example <delete-one-runnable-example>` returns 
+the following results:
+
+.. code-block:: none
+
+    Deleting a document with builders...
+    Deleted documents: 1
+
+.. tip:: Sample Datasets
+   
+   These examples use the :atlas:`sample datasets </sample-data>` provided by Atlas.
+   The number of documents returned may differ depending on the data in your
+   collection.
```

Comment:
This tip will not be necessary once the delete method is changed (per comment above)